### PR TITLE
Remove CSI and rplidar modules

### PR DESCRIPTION
**Pull request by sguequierre**
* Removing from docs site now that readmes are updated with configuration information

### CODE REVIEW

**Review by andf-viam — APPROVED** [review]
LGTM with two questions:

1. Left a comment regarding the `relatedcard` shortcode. Not sure how we want to proceed.
2. I'd like the team to think about and agree upon whether we:
- Link to modules using their Registry page, like https://app.viam.com/module/viam/rplidar
- Link to modules using their GitHub repo README page, like https://github.com/viamrobotics/csi-camera

I can see rationale for each!

**Review by npentrel — CHANGES_REQUESTED** [review]
related cards

File: docs/mobility/slam/_index.md
Comment (sguequierre) [inline docs/mobility/slam/_index.md:33]: @npentrel Took your rec for using external url with `alt_link` here but it is giving me a markdownlint error: no bare urls used

Comment (sguequierre) [inline docs/mobility/slam/_index.md:33]: @npentrel Took your rec for using external url with `alt_link` here but it is giving me a markdownlint error: no bare urls

Comment (sguequierre) [inline docs/mobility/slam/_index.md:33]:
```suggestion
{{< relatedcard link="/build/configure/components/camera/" alt_title="RPlidar" required="yes" alt_link=<"https://app.viam.com/module/viam/rplidar">>}}
```

Comment (npentrel) [reply docs/mobility/slam/_index.md:33]: Interesting. I don't know a way around it so let's turn off that rule. What's your thinking asking using app module URLs vs The github repo?

File: docs/mobility/slam/_index.md
Comment (npentrel) [inline docs/mobility/slam/_index.md:35]: Replace this with:
```suggestion
<div class="align-content-center">or</div>
```

And then add an item to the css file to do

```
.align-content-center {
    align-self: center
}
```

File: docs/mobility/slam/_index.md
Comment (npentrel) [reply docs/mobility/slam/_index.md:33]: I am still wondering why module URLs vs github repo - Andrew asked this too on the PR.

Comment (npentrel) [reply docs/mobility/slam/_index.md:33]: Also `{{< relatedcard link="ml/vision/vision-odometry/"  required="yes" alt_link="https://app.viam.com/module/viam/rplidar">}}` is not showing up. The link is not where that model lives.

**Review by npentrel — CHANGES_REQUESTED** [review]

File: docs/mobility/slam/_index.md
Comment (sguequierre) [reply docs/mobility/slam/_index.md:33]: @npentrel What do you mean/what can I do to turn off that rule?

Comment (sguequierre) [reply docs/mobility/slam/_index.md:33]: Also @npentrel I am going to link that card to GitHub as well as thatmodel page will be removed.

File: docs/mobility/slam/_index.md
Comment (sguequierre) [reply docs/mobility/slam/_index.md:35]: Done-- I think right css file, lmk if not

File: docs/mobility/slam/_index.md
Comment (sguequierre) [reply docs/mobility/slam/_index.md:33]: Wait, isn't this part inaccurate? In my understanding it wouldn't be Rplidar or Cartographer / Camera, it would be Rplidar / Cartographer. Want to confirm with you but should I change? @npentrel

Comment (npentrel) [reply docs/mobility/slam/_index.md]: I don't know a way to do this without triggering the markdownlint error. You can turn off the markdownlint rule in the .markdownlint settings. But it would be better to turn the rule off for this file only. Please check the markdownlint docs on how to do this. I don't know off the top of my head

Whether the module docs are on the docs site does not affect whether it's required for slam. If it is a requirement you should link to it.


@andf-viam said this about the two options in a previous comment. Probably worth asking the SME to confirm

File: docs/mobility/slam/_index.md
Comment (npentrel) [reply docs/mobility/slam/_index.md]: yep! all good!

**Comment by npentrel** [comment]
I think all good to go once https://github.com/viamrobotics/docs/pull/2279#discussion_r1415457299 is answered. I think you also need to rebase and fix the markdown formatting before merge

File: docs/mobility/slam/_index.md
Comment (sguequierre) [inline docs/mobility/slam/_index.md:30]:
```suggestion

<!-- markdownlint-disable MD034 -->

```

**Review by npentrel — APPROVED** [review]
Let's change the remaining app links to repo links and then good to merge

File: docs/registry/configure.md
Comment (sguequierre) [inline docs/registry/configure.md:389]:
```suggestion
   This example shows the configuration for adding a [CSI camera](https://github.com/viamrobotics/csi-camera/) as a local module.
```

File: docs/mobility/slam/cartographer/_index.md
Comment (sguequierre) [inline docs/mobility/slam/cartographer/_index.md:92]:
```suggestion
      - **Camera**: Select the `name` of the camera component that you created when you [added the `rplidar` module to your robot](https://github.com/viamrobotics/rplidar).
```

File: docs/mobility/slam/_index.md
Comment (sguequierre) [inline docs/mobility/slam/_index.md:25]:
```suggestion
You can conduct SLAM with data collected live by a [RPlidar](https://github.com/viamrobotics/rplidar) or with LIDAR data you provide in configuration, and easily view the map you build on the **SLAM library** tab of your location's page in the [Viam app](https://app.viam.com):
```

File: docs/mobility/slam/_index.md
Comment (sguequierre) [inline docs/mobility/slam/_index.md:34]:
```suggestion
{{< relatedcard link="/build/configure/components/camera/" alt_title="RPlidar" required="yes">}}
```

**Pull request merged**
